src/api/java/openiss/ws/soap/client/OpenISSSOAPClient.java

```diff
@@ -0,0 +1,40 @@
+package api.java.openiss.ws.soap.client;
+
+import api.java.openiss.ws.soap.service.OpenISSSOAPService;
+
+import java.io.IOException;
+import java.net.URL;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.namespace.QName;
+import javax.xml.ws.Service;
+
+@WebServlet(name = "OpenISSSOAPClient", urlPatterns = {"/OpenISSSOAPClient"})
+public class OpenISSSOAPClient extends HttpServlet {
```

Comment:
(B) Can we also please have a non-servlet plain Java client as well?

Reply:
can't find any solution for this review.